Do not pick up LLVM toolchain components (llvm-ar, llvm-ranlib)
which are only present in the version of LLVM in ports, not the
version in base.

Fix build with ports-gcc:
eg++: error: libgomp.spec: No such file or directory

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -422,7 +422,7 @@ ProcessorCount(num_cpus)
 # Compiler stuff
 
 if(CMAKE_C_COMPILER_ID STREQUAL Clang AND CMAKE_CXX_COMPILER_ID STREQUAL Clang)
-    include(LLVMToolchain)
+#    include(LLVMToolchain)
 endif()
 
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID STREQUAL Clang)
@@ -526,7 +526,6 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID S
             endif()
         endif()
 
-        set(MY_C_FLAGS ${MY_C_FLAGS} -fopenmp)
     endif()
 
     if(CMAKE_CXX_COMPILER_ID STREQUAL GNU)
